src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +49,30 @@
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get the historical price data for a given stock symbol.
+   *
+   * @param symbol stock symbol to get historical data for.
+   * @param range specified time range of the request.
+   * @param date specified date in the format YYYYMMDD.
+   * @return a list of historical price data for the given symbol, range, and date.
+   */
+  @GetMapping(value = "${mvc.iex.getHistoricalPricePath}", produces = {
+      MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrice> getHistoricalPrice(
+      @RequestParam(value = "symbol") final String symbol,
```

Comment:
What happens if the symbol is blank ("")?

Reply:
Same question for range and date

Reply:
I thought that it would respond with Bad Request since `symbol` is required by default, but I now see that one of my tests actually test for this case and get Not Found. I think it made sense to me when I was writing the tests, but I'll change this to test for empty `symbol` explicitly.

Reply:
I added checks!